bingo/bingo-elastic/python/bingo_elastic/model/record.py

```diff
@@ -17,7 +17,8 @@ def check_error(instance: IndigoRecord, error: BaseException) -> None:
     if instance.error_handler:
         instance.error_handler(instance, error)
     else:
-        raise error
+        pass
```

Comment:
why removing raising of the error? If no error handler is provided - we shouldn’t “eat” it

Reply:
Accidentally left here this "pass"( Thank you, brought everything back!